gfx.c

```diff
@@ -1025,6 +1027,18 @@ void ReadNtrCell_CEBK(unsigned char * restrict data, unsigned int blockOffset, u
             offset += 8;
         }
     }
+
+    if (options->ucatEnabled)
+    {
+        offset = blockOffset + 0x18 + ucatOffset + 0x04 * options->cellCount;
+
+        options->ucatCellAttribtes = malloc(sizeof(int) * options->cellCount);
```

Comment:
I'd personally prefer this one to be sizeof(uint32_t) or sizeof(int32_t)

Reply:
ensure that malloc'd data is freed where appropriate
usually in the options free